src/datetime.rs

```diff
@@ -1,13 +1,16 @@
 use super::WMIError;
-use chrono::prelude::*;
 use serde::{de, ser};
 use std::fmt;
 use std::str::FromStr;
+use time::format_description::{well_known::Rfc3339, FormatItem};
+use time::macros::format_description;
+use time::parsing::Parsed;
+use time::{PrimitiveDateTime, UtcOffset};
 
-/// A wrapper type around chrono's DateTime, which supports parsing from WMI-format strings.
+/// A wrapper type around time's OffsetDateTime, which supports parsing from WMI-format strings.
 ///
 #[derive(Debug)]
-pub struct WMIDateTime(pub DateTime<FixedOffset>);
+pub struct WMIDateTime(pub time::OffsetDateTime);
```

Comment:
Is there a way to make this a feature based change and support both?
Even better if we can make them not-mutually-exlusive.
We can reexport the chrono based one under the old name.

While chrono is a bit slow to react to that CVE, it's still very popular (and very good), and I think as a library wmi-rs should be less opinionated if possible 😄

Reply:
Done, behind a feature flag. I cannot use both at the same time without introducing generics in the type using `WMIDateTime` and a sealed trait pattern so I intentionally used a "replacement" feature instead of allowing both.